docs/decisions/0004-axios-caching-implementation.rst

```diff
@@ -0,0 +1,87 @@
+Axios Caching Layer Implementation
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The titans team noticed that on a lot of pages the same API calls are repeatedly being made as the user navigates around the website. The data being returned from these requests doesn't change between page loads leading to the same endpoints reloading the same data repeatedly. We decided that we wanted to avoid repeatedly reloading the data and reuse the data that was in the response from the initial load.
+
+We think it's beneficial to cut as much overhead as possible so each page becomes interactive more quickly.
+
+We decided to add a frontend caching layer to the AxiosJwtAuthService by leveraging an already existing npm package: `axios-cache-adapter <https://www.npmjs.com/package/axios-cache-adapter>`_
+
+Having a frontend caching layer would also allow for a stale version of a page to be displayed for a short amount of time in the case of a backend outage.
+
+When Frontend Caching Should Be Used
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The frontend cache should be used to eliminate the need to reload the same data on each page load as a user navigates around. It shouldn't be used to reduce the number of times multiple components on the same page try to make the same network call. Reducing the number of times the same network call is made on a single page should be done by moving the call higher up in the hierarchy so the call only gets made once and all lower level components are able to access the response data from it.
```

Comment:
Nit: "The frontend cache should be used to eliminate the need to reload the same data on each page load as a user navigates around" -> "The frontend cache should be used to eliminate the reloading of unchanged data as a user navigates between different pages in the application."